test/commands/login.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const cli = require('heroku-cli-util'),
+  cmd = require('../..').commands.find(c => c.topic === 'container' && c.command === 'login'),
+  expect = require('unexpected'),
+  sinon = require('sinon'),
+
+  Sanbashi = require('../../lib/sanbashi')
+var sandbox
+
+describe('container login', () => {
+  beforeEach(() => {
+    cli.mockConsole()
+    process.env.HEROKU_API_KEY = 'heroku_token'
+    sandbox = sinon.sandbox.create()
+  })
+  afterEach(() => sandbox.restore())
+
+  it('logs to the docker registry', () => {
```

Comment:
Should this also verify that the token is passed on standard in?

Reply:
You're right. Sinon is weird though. It doesn't fail when an argument is missing.